solarized-definitions.el

```diff
@@ -168,7 +168,7 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
               (fmt-bldi `(:weight ,bold                 :underline nil        :inverse-video nil))
               (fmt-undr `(:weight normal :slant normal  :underline ,underline :inverse-video nil))
               (fmt-undb `(:weight ,bold  :slant normal  :underline ,underline :inverse-video nil))
-              (fmt-undi `(:weight normal                :underline ,underline :inverse-video nil))
+              (fmt-undi `(:weight normal :slant ,italic :underline ,underline :inverse-video nil))
```

Comment:
Unfortunately (despite the apparent obviousness of this) the upstream theme doesn’t set italic on `fmt-undi`.

I’ll probably publish a variant of Solarized that does the obvious thing for `fmt-undi`, `fmt-bldi`, and a couple other things that don’t match.